### ISSUE TITLE
TIMIT Demo example

### ISSUE THREAD

**Issue by jedyang97** [freewym/espresso#28]
## 🚀 Feature Request

Would it be possible to upload an example for TIMIT for demonstration purpose? All other Speech Recognition datasets are kinda too large to download when just trying out this repo. Having TIMIT would make allow people new to ASR to quickly try out and appreciate the convinience of this framework. Thanks.

**Comment by freewym** [comment]
@jedyang97 Makes sense and thanks for your suggestion! I will write a recipe with a small enough dataset, like TIMIT, once I have time (these days I am too busy:().
